src/components/Settings/index.js

```diff
@@ -88,6 +91,16 @@ const Settings = ({
         <Switch isEnabled={shouldLiveSync} onToggle={handleShouldLiveSyncChange} />
       </div>
 
+      <div className="setting-container">
+        <div className="setting-label">
+          Sync on application becoming visible
+          <div className="setting-label__description">
+            If enabled, the current org file is pulled from to the sync backend when the browser tab becomes visible. This prevents you from having a stale file before starting to make changes to it.
```

Comment:
Minor typo here: `...from to the sync...` should be `...from the sync...`

Reply:
Good catch. Thanks!

Done.